src/pages/interim/_love-trading.js

```diff
@@ -80,18 +85,93 @@ const MobileWrapper = styled.div`
     }
 `
 
-const MarLeft = styled(Flex)`
-    margin-left: 0;
-    max-width: 38.4rem;
-`
-
 const FitButton = styled(LinkButton)`
     width: fit-content;
 `
 
-const White = styled(Header)`
-    color: var(--color-white);
-    width: 38.4rem;
+const query = graphql`
+    query {
+        affiliate: file(relativePath: { eq: "affiliate.png" }) {
+            ...fadeIn
+        }
+        smart_trader: file(relativePath: { eq: "smarttrader.png" }) {
+            ...fadeIn
+        }
+        dmt5: file(relativePath: { eq: "interim-dmt5.png" }) {
+            ...fadeIn
+        }
+        dbot: file(relativePath: { eq: "interim-dbot.png" }) {
+            ...fadeIn
+        }
+    }
 `
 
-export { Section, Responsive, AbsoluteWrapper, MarLeft, FitButton, White, MobileWrapper }
+const LeftComponent = ({ params }) => {
```

Comment:
Can we give more descriptive name rather than `LeftComponent` and `RightComponent`